numba/array_analysis.py

```diff
@@ -825,6 +828,31 @@ def _get_shape(self, obj):
             else:
                 return super(SymbolicEquivSet, self)._get_shape(obj)
 
+class WrapIndexMeta(object):
+    """
+      Array analysis should be able to analyze all the function
+      calls that it adds to the IR.  That way, array analysis can
+      be run as often as needed and you should get the same
+      equivalencies.  A difficult call to analyze that array
+      analysis will add is to wrap_index.  The important thing to
```

Comment:
Not sure what this sentence means: `A difficult call to analyze that array analysis will add is to wrap_index.`?

Reply:
Tried to clarify.